Tests/LibUnicode/TestIDNA.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, Simon Wanner <simon@skyrising.xyz>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <LibUnicode/IDNA.h>
+
+namespace Unicode::IDNA {
+
+TEST_CASE(to_ascii)
```

Comment:
Same comment here for tests, should be in the same commit that added the function being tested